templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
I've completed user tests with the following feedback: https://github.com/neomad-team/neomad.org/pull/115#issuecomment-324452994